### PR TITLE
Handle article image

### PR DESCRIPTION
**Pull request by blaisefonseca**

### CODE REVIEW

File: app/services/article-service.js [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
             function query(qs){
                 
-                return $q.when($http.get('/api/v2017/articles', { params: qs, cache:true}))
+                return $q.when($http.get('/api/v2017/articles', { params: qs}))
```
Comment (sbilodeau) [inline app/services/article-service.js]: whay wrapeing you $http call with `$q.when()`

File: app/services/article-service.js [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
                         .then(function(data){
                             return data.data;
                         });
                 
             }
             function get(articleId){
                 
-                return $q.when($http.get('/api/v2017/articles/' + encodeURIComponent(articleId), {cache:true}))
+                return $q.when($http.get('/api/v2017/articles/' + encodeURIComponent(articleId)))
```
Comment (sbilodeau) [inline app/services/article-service.js]: same a line 7

**Comment by sbilodeau** [comment]
You are mixing concept. upgrade to CND and article image handling!

File: app/services/article-service.js (hunk 1 above)
Comment (blaisefonseca) [reply app/services/article-service.js]: removed!

File: app/services/article-service.js (hunk 2 above)
Comment (blaisefonseca) [reply app/services/article-service.js]: removed!

File: app/directives/articles/cbd-article.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ define(['app', 'text!./cbd-article.html','lodash', 'services/article-service', '
 			link: function ($scope, $element, $attr)
 			{
 				$scope.hideCoverImage = $attr.hideCoverImage||false;
+				$scope.returnUrl	  = $location.$$absUrl;
```
[the remark this answers is not in the document]
Comment (blaisefonseca) [reply app/directives/articles/cbd-article.js]: done!

**Comment by blaisefonseca** [comment]
> You are mixing concept. upgrade to CND and article image handling!

yes, i will try to create new PR in future

**Pull request merged**